src/main/java/com/bobocode/nasapicturesstealer/persistance/entity/Camera.java

```diff
@@ -0,0 +1,40 @@
+package com.bobocode.nasapicturesstealer.persistance.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+import static jakarta.persistence.GenerationType.IDENTITY;
+
+@Builder
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "cameras", schema = "public")
+public final class Camera {
```

Comment:
Entity should not be final. Key reasons:
 Proxying: Hibernate uses proxies for lazy loading. It creates a proxy subclass of an entity to manage lazy loading. If the entity class is final, Hibernate can't create a proxy subclass, which inhibits lazy loading.Dynamic Bytecode Generation: Hibernate utilizes dynamic bytecode generation for various optimizations, like dirty checking. This process requires subclassing the entity, which isn't possible if the entity is final.Flexibility: Non-final entities allow for more flexibility in the framework's operations, including enhancements and various optimizations that might be needed during runtime.

Reply:
Got it, thanks

Reply:
Done